### ISSUE TITLE
2020-1 OSS miniproject 구현 및 테스트

### ISSUE THREAD

**Issue by TaeyoungLee10** [lhb0927/2020OSS_miniproject#10]
여기에서 각 함수들에 대한 구현과 테스트에 대한 내용을 상의하면 좋을 것 같습니다~

**Comment by lhb0927** [comment]
네! 이번주는 검색기능 제외하고 나머지 함수들 진행하면 좋을 것 같습니다!

**Comment by TaeyoungLee10** [comment]
저희 둘다 저번에 구현 해놓은 상태라서, 각자 다시 테스트 해보면서 더 수정할 사항 있으면 보면 좋을것 같아요~
저는 우선 저번에 구현한 listmenu함수에서 메뉴 이름이 길어지면 출력이 깔끔하게 나오지 않는 것 같아서 그 부분을 다시 수정해보겠습니다.

학우님께서 구현하신 저장, 불러오기, 검색함수도 한번 테스트해봐서 피드백 보내드릴게요 :)

**Comment by TaeyoungLee10** [comment]
학우님도 제가 수정하거나 추가하면 좋을 것 같은 부분있으면 피드백 언제든지 주세요 :)

**Comment by lhb0927** [comment]
네, 제가 맡은 함수들 중 파일 저장하는 함수는 메뉴의 이름/가격/크기/타입 순으로 한 줄씩 저장하도록 구현 해두었습니다. 저장된 파일을 불러오는 함수는 아직 완성되지 않아서 완성하는데로 남겨두겠습니다.

**Comment by lhb0927** [comment]
그리고 테스트 해보고 수정이 필요하다고 생각하는 사항 남겨두겠습니다.

**Comment by lhb0927** [comment]
교수님이 말씀하긴 대로 꾸준히 조금씩 진행하는게 좋을거 같아요 :)
저는 main.c 에서 함수 실행할 때에 가독성이 좀 좋도록 약간 수정해 두었습니다.

**Comment by TaeyoungLee10** [comment]
감사합니다!
저도 조금씩 수정해보도록 할게요~ 수정 끝날 때마다 issue에 댓글 달도록 하겠습니다

**Comment by lhb0927** [comment]
loadData function 수정해서 테스 완료했습니다 :)
간단하게 CRUD 실행 및 파일 저장/로드를 실행하는데에 큰 오류는 없어보입니다.
가독성이나 다른 예외처리 같은것만 수정하면 될것 같네요!

**Comment by TaeyoungLee10** [comment]
알겠습니다! 저도 수정하는데로 올리겠습니다~

**Comment by TaeyoungLee10** [comment]
product.c에서 메뉴의 목록을 조회할때 메뉴의 글자수에 상관없이 깔끔하게 출력되도록 수정했습니다.

**Comment by TaeyoungLee10** [comment]
프로그램에 대해 간단히 설명해주는 함수를 추가해봤는데, 꼭 필요한 기능은 아니긴 해서 한번 보시고 추가해도 괜찮을지 말씀해주실 수 있나요?

**Comment by lhb0927** [comment]
확인해보겠습니다~

**Comment by lhb0927** [comment]
추가해도 괜찮을거 같아요! 거의 완성 단계라 저도 뭘 더 추가해야할지 고민을 해봐야겠네요..

**Comment by TaeyoungLee10** [comment]
오오 네넵! 제가 그래도 생각났던 추가적인 기능들은 S,M,L을 잘 입력했는지 검사해주고 다시 입력을 요구하는 함수랑, ICE랑 HOT이 잘 입력되었는지 확인해주는 함수였습니다.
이 부분 혹시 추가해보는거 어떠세요?

**Comment by lhb0927** [comment]
아무래도 이상한 값을 입력 받을수도 있으니 추가해 보는게 좋을거 같네요!

**Comment by TaeyoungLee10** [comment]
알겠습니다! 저는 그러면 S,M,L을 정상적으로 받아오도록 하는 함수를 구현해보도록 하겠습니다.

**Comment by lhb0927** [comment]
그럼 제가 내일 ICE랑 HOT 입력 받는 부분 수정하겠습니다.
